src/background_music/background_music.cc

```diff
@@ -0,0 +1,49 @@
+/*
+ * Background music (equal loudness) Plugin for Audacious
+ * Copyright 2023 Michel Fleur
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions, and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions, and the following disclaimer in the documentation
+ *    provided with the distribution.
+ *
+ * This software is provided "as is" and without any warranty, express or
+ * implied. In no event shall the authors be liable for any damages arising from
+ * the use of this software.
+ */
+#include "FrameBasedEffectPlugin.h"
+#include <libaudcore/i18n.h>
+#include <libaudcore/preferences.h>
+
+static constexpr const PreferencesWidget background_music_widgets[] = {
+    WidgetLabel(N_("<b>Background music</b>")),
+    WidgetSpin(N_("Target level (dB)"),
+               WidgetFloat(CONFIG_SECTION_BACKGROUND_MUSIC,
+                           CONF_TARGET_LEVEL_VARIABLE),
+               {CONF_TARGET_LEVEL_MIN, CONF_TARGET_LEVEL_MAX, 1.0}),
+    WidgetSpin(N_("Maximum amplification"),
+               WidgetFloat(CONFIG_SECTION_BACKGROUND_MUSIC,
+                           CONF_MAX_AMPLIFICATION_VARIABLE),
+               {CONF_MAX_AMPLIFICATION_MIN, CONF_MAX_AMPLIFICATION_MAX, 1.0}),
+    WidgetLabel(N_("<b>Advanced</b>")),
+    WidgetSpin(
+        N_("Detection Balance\n(-1=perception; 1=slow)"),
+        WidgetFloat(CONFIG_SECTION_BACKGROUND_MUSIC, CONF_BALANCE_VARIABLE),
+        {CONF_BALANCE_MIN, CONF_BALANCE_MAX, 0.1})};
```

Comment:
- Move the unit (like dB) behind the spin button
- Add a colon to the spin button labels
- Please also describe better what the balance actually influences, for example in a separate label.

My suggestion to get the idea:

```diff
diff --git a/src/background_music/background_music.cc b/src/background_music/background_music.cc
index 33b840e4b..8cf0d2b03 100644
--- a/src/background_music/background_music.cc
+++ b/src/background_music/background_music.cc
@@ -22,19 +22,21 @@
 
 static constexpr const PreferencesWidget background_music_widgets[] = {
     WidgetLabel(N_("<b>Background music</b>")),
-    WidgetSpin(N_("Target level (dB)"),
+    WidgetSpin(N_("Target level:"),
                WidgetFloat(CONFIG_SECTION_BACKGROUND_MUSIC,
                            CONF_TARGET_LEVEL_VARIABLE),
-               {CONF_TARGET_LEVEL_MIN, CONF_TARGET_LEVEL_MAX, 1.0}),
-    WidgetSpin(N_("Maximum amplification"),
+               {CONF_TARGET_LEVEL_MIN, CONF_TARGET_LEVEL_MAX, 1.0, N_("dB")}),
+    WidgetSpin(N_("Maximum amplification:"),
                WidgetFloat(CONFIG_SECTION_BACKGROUND_MUSIC,
                            CONF_MAX_AMPLIFICATION_VARIABLE),
                {CONF_MAX_AMPLIFICATION_MIN, CONF_MAX_AMPLIFICATION_MAX, 1.0}),
     WidgetLabel(N_("<b>Advanced</b>")),
     WidgetSpin(
-        N_("Detection Balance\n(-1=perception; 1=slow)"),
+        N_("Detection Balance:"),
         WidgetFloat(CONFIG_SECTION_BACKGROUND_MUSIC, CONF_BALANCE_VARIABLE),
-        {CONF_BALANCE_MIN, CONF_BALANCE_MAX, 0.1})};
+        {CONF_BALANCE_MIN, CONF_BALANCE_MAX, 0.1}),
+    WidgetLabel (N_("<b>Hint</b>")),
+    WidgetLabel(N_("Use a balance of -1 to [...]\nHigher values cause [...]\nFor [...] use a value of 1."))};
 
 static constexpr const PluginPreferences background_music_preferences = {
     {background_music_widgets}};
```

Reply:
Fixed [here](https://github.com/audacious-media-player/audacious-plugins/commit/cc4ae72111bcc55bde3c1eed6121efdafd11a629)